set.mm

```diff
@@ -12587,6 +12587,201 @@ simplest antecedents (i.e., in the corresponding ordering of binary trees
     ( wi minimp-ax2 minimp-ax1 ax-mp mp2 ) AABCZCZAACZHCCIJCZIHCAABDAHACCKAHEAH
     ADFIJHDG $.
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Implicational Calculus
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+  Implicational calculus is the fragment of propositional logic that uses only
+  material implication, and not negation. It can be axiomatized by inference
+  rule modus ponens ~ ax-mp together with the axioms { ~ ax-1 , ~ ax-2 ,
+  ~ peirce } or the Tarski-Bernays axioms { ~ ax-1 , ~ imim1 , ~ peirce } or
+  with the single axiom { ~ impsingle }. From either one of these three axiom
+  sets, all tautologies containing only material implication may be proved. In
+  contrast, minimal implicational calculus, which is presented just before this
+  section, cannot prove certain tautologies ( ~ peirce , for example ). The
+  class of theorems proved by minimal implicational calculus is thus a subset
+  of the class of theorems proved by implicational calculus.
+
+  The primary source for this section is the paper by Lukasiewicz, _The
+  Shortest Axiom of the Implicational Calculus of Propositions_, Proceedings
+  of the Royal Irish Academy, Section A, vol. 52 (1948-1950), 25--33. It will
+  be cited as simply "Lukasiewicz" in the remainder of this section.
+
+  This section proves that the above three distinct axiom sets for
+  implicational calculus all produce the same class of theorems.
+
+  (Contributed by Larry Lesyna and Jeffrey P. Machado, 1-Aug-2023.)
+
+$)
+
+
+  $( The shortest single axiom for implicational calculus, due to Lukasiewicz.
+     It is now known to be the _unique_ shortest axiom.  The axiom is proved
+     here starting from { ~ ax-1 , ~ ax-2 and ~ peirce }.  (Contributed by
+     Larry Lesyna and Jeffrey P. Machado, 18-Jul-2023.)
+     (Proof modification is discouraged.)  (New usage is discouraged.) $)
+  impsingle $p |- ( ( ( ph -> ps ) -> ch )
+     -> ( ( ch -> ph ) -> ( th -> ph ) ) ) $=
+    ( wi imim1 peirce a1d syl6 ) ABEZCECAEJAEZDAEJCAFKADABGHI $.
+
+  $( Derivation of impsingle-step4 from ~ ax-mp and ~ impsingle .  It is used
+     as a lemma in proofs of ~ imim1 and ~ peirce from ~ impsingle .  It is
+     Step 4 in Lukasiewicz, where it appears as 'CCCpqpCsp' using
+     parenthesis-free prefix notation.  (Contributed by Larry Lesyna and
+     Jeffrey P. Machado, 2-Aug-2023.)  (Proof modification is discouraged.)
+     (New usage is discouraged.)) $)
```

Comment:
```suggestion
     (New usage is discouraged.) $)
```

same for impsingle-step8, 15, 18, 19, 20, 21, 22

Reply:
Done.